src/collections/GenericCollection.tsx

```diff
@@ -9,7 +9,8 @@ import { Multiline } from "../components/Multiline";
 export const GenericCollection: React.FC<{
     collection: ICollection;
 }> = ({ collection }) => {
-    if (collection.backfill.length < 1) {
+    const content = [].concat(collection.curated).concat(collection.backfill);
```

Comment:
Think you can just do:

    [].contact(a, b)

i.e. concat is multi-arity.

Reply:
Yeah of course 🤦‍♂️